src/frontend/typer/søknad.ts

```diff
@@ -152,6 +153,11 @@ export const initialStateSøknad: ISøknad = {
             svar: '',
         },
         arbeidsperioderUtland: [],
+
+        arbeiderINorge: {
+            id: EøsSøkerSpørsmålId.arbeiderINorge,
+            svar: null,
+        },
```

Comment:
Er det riktig at den skal ligge her?

Reply:
Ja 👏 men vi må ta høyde for enke her og

Reply:
Nei nå surrer jeg, en søker er jo ikke død 😂 se bort i fra enke